src/components/elements/popover/types.ts

```diff
@@ -1,9 +1,17 @@
 import { PopoverInlineProps } from "@lifesg/react-design-system/popover-v2";
 import * as Icons from "@lifesg/react-icons";
+import { ReactElement } from "react";
 import { IBaseElementSchema } from "../types";
 
+export enum PopoverHintType {
+	DEFAULT = "default",
+	IMAGE = "image",
+	COMPONENT = "component",
+}
+
 interface IPopoverHint extends Pick<PopoverInlineProps, "position" | "customOffset" | "zIndex"> {
-	content: string;
+	content: string | ReactElement;
```

Comment:
we should aim to keep elements serializable. using ReactElement is possible for custom apps with access to source code but not for JSON-only config

there are two options:
1. update `Sanitize` usage in popover to support img tags (you can refer to `src/components/elements/text/text.tsx`)
2. allow `content` to accept schema object for more control (you can refer to `topSection` and `bottomSection` of IReviewBaseSchema in `src/components/custom/review/types.ts`)

option 2 is more flexible and will likely be more suitable for the requestor's usecase since they seem to need a more complex layout

Reply:
while we're here, can we check if `Sanitize` preserves the `alt` attribute? this is likely important for accessibility